packages/sampler/stories/qa-stories/VideoPlayer.js

```diff
@@ -61,15 +59,12 @@ class VideoSourceSwap extends React.Component {
 				{button('Next Preload Video without changing preload', this.nextVideoKeepPreload)}
 				{button('Change Preload without changing video', this.nextPreloadVideoKeepVideo)}
 				{button('Reset Sources', this.resetSources)}
-				<VideoPlayer
-					muted
-					preloadSource={<source src={this.state.playlist[this.state.preloadCursor]} type="video/mp4" />}
-				>
-					<source src={this.state.playlist[this.state.cursor]} type="video/mp4" />
+				<VideoPlayer muted>
```

Comment:
It's declared on `VideoPlayer` and included (by not explicitly removing it) in `mediaProps` so this should work. It would _also_ work being declared on `Video` though since `VideoPlayer` doesn't explicitly handle it in any way.

Reply:
Yeah. I was hoping to remove `Media`/`Video` specific props from `VideoPlayer` if possible. I have a similar concern with `noAutoPlay` prop, too. But `noAutoPlay` affects the initial value of one of VP's internal member currently, so I'm not super concerned to fix that in this PR.

It would look something like this when changed:
```
<VideoPlayer>
  <Video muted noAutoPlay />
</VideoPlayer